src/pages/devices/devices.tsx

```
@@ -0,0 +1,143 @@
+    return () => {
+      if (pageWrapper.current) {
+        resizeObserver.unobserve(pageWrapper.current);
+      }
+    };
+  }, [pageWrapper, pageWidth]);
+
+  useEffect(() => {
+    splitDevicesIntoColumns();
+  }, [store.filteredDevices.size]);
+
+  const isWithFractionalValues = useCallback((cells) => {
```

ESLint

[error] 'isWithFractionalValues' is assigned a value but never used. Allowed unused vars must match /^(_|[A-Z])[a-zA-Z0-9]*$/u.

(no-unused-vars)